rdf2hpcc/src/main/java/org/hpccsystems/rdf/rdf2hpcc/RDFHPCCWsClient.java

```diff
@@ -821,41 +818,32 @@ private boolean setDropZoneInfo() throws Exception
 
         HPCCFileSprayClient fsclient = getFileSprayClient();
 
-        DropZoneFilesRequest dropzonefilesreq = new DropZoneFilesRequest();
-
         try
         {
-            DropZoneFilesResponse dropZoneFilesResponse = fsclient.getSoapProxy().dropZoneFiles(dropzonefilesreq);
+            DropZone[] dropZones = fsclient.fetchLocalDropZones();
 
-            if (dropZoneFilesResponse.getExceptions() != null)
-            {
-                Utils.println(System.out, "Failed to fetch dropzone IP address.", false, verbosemode);
-            }
-            else
+            if (dropZones.length > 0)
             {
-                DropZone[] dropZones = dropZoneFilesResponse.getDropZones();
-                if (dropZones.length > 0)
-                {
-                    targetDropzoneNetAddres = dropZones[0].getNetAddress();
-                    targetHPCCDropzonePath = dropZones[0].getPath();
-                    Utils.println(System.out, "Found dropzone net address: " + targetDropzoneNetAddres, false, verbosemode);
-                    Utils.println(System.out, "Found dropzone path: " + targetHPCCDropzonePath, false, verbosemode);
+                targetDropzoneNetAddres = dropZones[0].getNetAddress();
```

Comment:
targetDropzoneNetAddres is missing an s on the end.  It's consistent throughout the file but I'd rather it be spelled correctly.